tests/02_00_06-2wp-new-minimum.js

```diff
@@ -1,117 +1,130 @@
+        const recipientRskAddressBalanceInitial = Number(await rskTxHelper.getBalance(recipientRskAddressInfo.address));
+        expect(recipientRskAddressBalanceInitial).to.equal(0);
 
+        await btcTxHelper.fundAddress(senderAddressInfo.address, MINIMUM_PEGIN_VALUE_IN_BTC + btcTxHelper.getFee());
+        
         // Execute peg-in
-        const peginBtcTxHash = await _2wpUtilsLegacy.sendPegin(rskClient, btcClient, senderAddressInformation, AMOUNT_TO_LOCK_IN_BTC);
-        await _2wpUtilsLegacy.ensurePeginIsRegistered(rskClient, peginBtcTxHash);
+        const btcPeginTxHash = await sendPegin(rskTxHelper, btcTxHelper, senderAddressInfo, MINIMUM_PEGIN_VALUE_IN_BTC);
+        await ensurePeginIsRegistered(rskTxHelper, btcPeginTxHash);
+
         // Assert
-        let finalSenderBalance = await btcClient.getAddressBalance(senderAddressInformation.address);
-        let finalDerivedAddressBalance = await rskClient.eth.getBalance(rskDerivedAddress);
+        const senderAddressBalanceFinal = Number(await btcTxHelper.getAddressBalance(senderAddressInfo.address));
+        expect(senderAddressBalanceFinal).to.equal(0);
 
-        expect(Number(initialSenderBalance)).to.equal(0);
-        expect(Number(finalSenderBalance)).to.equal(0);
-        expect(Number(initialDerivedAddressBalance)).to.equal(0);
-        expect(Number(finalDerivedAddressBalance)).to.equal(rsk.btcToWeis(AMOUNT_TO_LOCK_IN_BTC));
+        const recipientRskAddressBalanceFinal = Number(await rskTxHelper.getBalance(recipientRskAddressInfo.address));
+        expect(recipientRskAddressBalanceFinal).to.equal(btcEthUnitConverter.btcToWeis(MINIMUM_PEGIN_VALUE_IN_BTC));
     });
 
-    it('should not peg-in when sending below minimum value', async () => {
-        const AMOUNT_TO_LOCK_IN_BTC = btcClient.nodeClient.satoshisToBtc(minimumPeginValueInSatoshis - 1);
+    it('should not peg-in and not refund when sending below minimum value', async () => {
+        const PEGIN_VALUE_IN_BTC = MINIMUM_PEGIN_VALUE_IN_BTC - 0.001
 
         // Create legacy type address to use as sender
-        let senderAddressInformation = await btcClient.generateBtcAddress('legacy');
-        let initialSenderBalance = await btcClient.getAddressBalance(senderAddressInformation.address);
+        const senderAddressInfo = await btcTxHelper.generateBtcAddress('legacy');
 
-        // Get the RSK address where the funds should be locked to
-        let rskDerivedAddress = keyUtils.getRskAddress(senderAddressInformation.privateKey);
-        let initialDerivedAddressBalance = await rskClient.eth.getBalance(rskDerivedAddress);
+        await btcTxHelper.fundAddress(senderAddressInfo.address, PEGIN_VALUE_IN_BTC + btcTxHelper.getFee());
 
-        await btcClient.fundAddress(senderAddressInformation.address, AMOUNT_TO_LOCK_IN_BTC + btcClient.getFee());
+        //get federation address
+        const federationAddress = await bridge.methods.getFederationAddress().call();
```

Comment:
Move This To The `before` Method And Use For The 3 Tests Cases

Reply:
i use it only in one test case...